dev/com.ibm.ws.request.timing_fat/fat/src/com/ibm/ws/request/timing/fat/TimingRequestTiming.java

```diff
@@ -219,7 +219,7 @@ public void testDynamicTimingEnableDisable() throws Exception {
         server.waitForStringInLogUsingMark("TRAS0114W", 10000);
         int n_hung = fetchHungRequestWarningsCount();
 
-        assertTrue("Expected > 0 slow request warning but found : " + n_slow, ((n_slow - slow) > 0));
+        assertTrue("Expected > 0 slow request warning but found : " + (n_slow - slow), ((n_slow - slow) > 0));
         assertTrue("Expected 0 hung request warning but found : " + n_hung, ((n_hung - hung) == 0));
```

Comment:
Can you fix the value in the message to match the condition too?

Reply:
Fixed